code/modules/mob/living/silicon/robot/robot_subtypes/dummy.dm

```diff
@@ -0,0 +1,15 @@
+/mob/living/silicon/robot/dummy
+	name = "cybernetic target dummy"
+	desc = "Great for practicing on."
+	cell_type = /obj/item/weapon/cell/crap
+	anchored = TRUE
+	scrambledcodes = TRUE
```

Comment:
most of these vars get set by the UnlinkSelf call in New, what's the point of this?

Reply:
i thought i removed the `UnlinkSelf()` call, that proc isn't necessary when these vars are set up right
saves a cpu cycle

Reply:
but unlinkself also removes the borg from the camera network

Reply:
scrambledcodes makes it so it never gets connected to the network in the first place